recipes/leptonica/all/conanfile.py

```diff
@@ -73,12 +73,15 @@ def requirements(self):
             self.requires("zlib/1.2.12")
         if self.options.with_gif:
             self.requires("giflib/5.2.1")
-        if self.options.with_jpeg:
-            self.requires("libjpeg/9d")
+        if self.options.with_jpeg == "libjpeg":
+            self.requires("libjpeg/9e")
+        if self.options.with_jpeg == "libjpeg-turbo":
+            self.requires("libjpeg-turbo/2.1.4")
         if self.options.with_png:
             self.requires("libpng/1.6.38")
         if self.options.with_tiff:
             self.requires("libtiff/4.4.0")
+            self.options["libtiff"].jpeg = self.options.with_jpeg
```

Comment:
not here, in `configure()`

Reply:
Nevertheless it seems to come through correctly. I tested at locally and did it once again here just to ensure. The package test here prints the versions of the dependencies and it is already printing out:

```
Leptonica version: 1.82.0
Lib versions: libgif 5.2.1 : libjpeg 8d (libjpeg-turbo 2.1.4) : libpng 1.6.38 : libtiff 4.4.0 : zlib 1.2.12 : libwebp 1.2.4 : libopenjp2 2.5.0
```

I will open an PR just to clean it up and move it into the config section.